test/appsignal/nif_test.exs

```diff
@@ -14,7 +14,16 @@ defmodule Appsignal.NifTest do
     assert is_binary(transaction)
   end
 
-  test "the nif is loaded" do
-    assert true = Appsignal.Nif.loaded?
+  if Mix.env in [:test, :test_phoenix] do
```

Comment:
wouldn't `if Mix.env != :test_no_nif do` be more flexible if we add more test suites than `test` and `test_phoenix`?